File: Projeto_7/src/models/Usuario.py
from src.models.TipoUsuario import TipoUsuario

class Usuario():
    def __init__(self, nome='indefinido', email='indefinido', tipo=TipoUsuario.DEFAULT):
        # Construtor único e parametrizado com valores padrão
        self._nome = nome
        self._email = email
        self._tipo = tipo
    

File: Projeto_6/Projeto_6 - Alissa Garcia/src/Usuario.cpp
#include "Usuario.h"

Usuario::Usuario() {
    nome = "NULL";
    email = "NULL";
    tipo = TipoUsuario::ALUNO;
}

Usuario::Usuario(string nome, string email, TipoUsuario tipo) {
    this->nome = nome;
    this->email = email;
    this->tipo = tipo;
}
